lib/order.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'menu'
+
+attr_reader :menu
+
+class Order
+  def initialize menu
+    @menu = menu
+  end
+
+  def place_order(*dish_number)
```

Comment:
Unused method argument - `dish_number`. If it's necessary, use `_` or `_dish_number` as an argument name to indicate that it won't be used. You can also write as `place_order(*)` if you want the method to accept any arguments but don't care about them.